src/UCTSearch.cpp

```diff
@@ -502,6 +502,8 @@ bool UCTSearch::should_halt_search() {
 // Asks the search to stop politely
 void UCTSearch::please_stop()
 {
+    Limits.infinite = 0;
```

Comment:
I think my PR is a better approach to this.  Feel free to merge it in to yours.

Reply:
https://github.com/glinscott/leela-chess/pull/294/files This one right?

Reply:
yeah